pgrx/src/datum/json.rs

```diff
@@ -122,7 +122,7 @@ impl FromDatum for JsonString {
 /// for json
 impl IntoDatum for Json {
     fn into_datum(self) -> Option<pg_sys::Datum> {
-        let string = serde_json::to_string(&self.0).expect("failed to serialize Json value");
+        let string = serde_json::to_string(&self.0).expect("failed to serialize a json value");
```

Comment:
reframe in terms of requirements, etc.

Reply:
An attempt to find a correct wording has been failed completely. The thing here is that `serde_json` should be able to serialize it's own types without any issue. Moreover, the `Value` type says that it's a valid JSON representation.

Reply:
Ah. Then if we don't have anything useful to say, simply `.unwrap()`.